features/admin.applications.v1/components/settings/sign-on-methods/script-based-flow/script-based-flow.tsx

```diff
@@ -1292,7 +1279,7 @@ export const ScriptBasedFlow: FunctionComponent<AdaptiveScriptsPropsInterface> =
                             <>
                                 <div className="conditional-auth-accordion-title">
                                     {
-                                        !readOnly && !isScriptUpdateReadOnly && (
+                                        !isPremiumOrReadOnly && !isScriptUpdateReadOnly && (
```

Comment:
It would be better if we could use one common variable to define if the script is read-only or not.